src/kvstore/RocksEngine.cpp

```diff
@@ -425,5 +430,48 @@ ResultCode RocksEngine::flush() {
     }
 }
 
+ResultCode RocksEngine::createCheckpoint(const std::string& path) {
+    LOG(INFO) << "Begin checkpoint : " << dataPath_;
+    auto checkpointPath = folly::stringPrintf(checkpointPath_.c_str(), path.c_str());
+    LOG(INFO) << "Target checkpoint path : " << checkpointPath;
+    if (fs::FileUtils::exist(checkpointPath)) {
```

Comment:
Do we need to handle when checkpointPath is empty?

Reply:
checkpointPath was removed. The default checkpoint already bound by data path.